locale/fr.po

```diff
@@ -532,14 +532,13 @@ msgstr "[c] git checkout -- '*'"
 msgid "[d] delete build dir and try again"
 msgstr "[d] effacer le répertoire de compilation et réessayer"
 
-#: pikaur/install_cli.py:208 pikaur/build.py:703
+#: pikaur/build.py:703 pikaur/install_cli.py:208
 msgid "[e] edit PKGBUILD"
 msgstr "[e] éditer le PKGBUILD"
 
 #: pikaur/build.py:700
-#, fuzzy
 msgid "[f] check() skip"
-msgstr "[c] sauter les sommes de contrôle"
+msgstr "[f] vérification() ignorée"
```

Comment:
same here

Reply:
Hi,
how should I translate this line? 
- do not change anything ("[f] check() skip")
- or "[f] ignorer check()"

Reply:
i'm not sure if original english string is clear enough

Reply:
Ok,
I'm waiting for your modification.

Reply:
@Bundy01 what the output of ` makepkg --help | grep '\-\-nocheck'` in your languages?

Reply:
@actionless: 
Ne pas exécuter la fonction check() dans PKGBUILD
Do not execute the check() function in PKGBUILD (translated from the original French message into English)

Reply:
done!